examples/ffdebye/many_beads/data.hessian

```diff
@@ -0,0 +1,10 @@
+# Hessian matrix (atomic units)
```

Comment:
In all other files the syntax for the units comes inside brackets instead of parenthesis. Can you change that to make it uniform?

Reply:
I'm not sure what you mean, but I see in the examples other files like this one. 
E.g. examples/lammps/h2o-vscf/dynmat.data, examples/ffdebye/single_particle/data.hessian

Note that it's not a machine-read line, just a comment. Maybe I can just erase it to avoid misleading.

Reply:
I added explanation to the helpline. We can think about changing the way how such auxiliary files are imported (right now it's basically Numpy arrays, comments are ignored), but for now I would leave it as it is.